contigious array.java

```diff
@@ -0,0 +1,44 @@
+// Time Complexity :o(n).
```

Comment:
try using only one pointer to avoid confusion, consider 0 as -1 and calculate running sum